aws-lambda-function/variables.tf

```diff
@@ -42,7 +42,7 @@ variable runtime {
 
 variable timeout {
   type        = number
-  description = "Execution timeout for the lambda."
+  description = "Execution timeout for the lambda. If null, defaults to 3 sec."
```

Comment:
I think we should avoid making this statement in our docs, as we don't implement it.

Reply:
Instead you could say something like "defaults to AWS default".